eng/CredScanSuppression.json

```diff
@@ -1,4 +1,21 @@
 {
-    "tool": "Credential Scanner",
-    "suppressions": []
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
```

Comment:
If we want to suppress the known hash value, we can also use the following rule format:
```
{
"tool": "Credential Scanner",
"suppressions": [
{
"placeholder": "$CREDENTIAL_PLACEHOLDER$",
"_justification": "This is the secret place holder used by my team"
}
]
}
```